README.adoc

```diff
@@ -119,10 +119,9 @@ Navigate back to the `start` directory to begin.
 
 == Adding health checks to microservices
 
-Navigate to the `pom.xml` file to check the required dependency. The `microprofile-health-api`
-dependency is located in the `pom.xml` file. With this dependency, you can
-use the MicroProfile Health API to provide health checks to your microservices.
-The `mpHealth-1.0` feature is enabled in the `src/main/liberty/config/server.xml` file.
+Navigate to the `pom.xml` file to check the required dependency. The `microprofile-health-api` dependency is added so you
```

Comment:
I think @sishida would like this to be: 
"The MicroProfile Health API was added as a dependency to your pom.xml file. Look for the dependency with the microprofile-health-api group ID. This feature allows you to use the MicroProfile Health API to provide health checks for your microservices."

This will also be consistent with the other guides.

Reply:
Fixed